### ISSUE TITLE
Field description

### ISSUE THREAD

**Issue by PautassoEGiacomina** [sqlitebrowser/sqlitebrowser#2709]
### Description
I tried to search if anyone had already proposed this, but I couldn't find anything. In case this has already been mentioned, please excuse me and delete the thread.
It would be veeery nice if there was the possibility to add a description to the various fields we create, so as to make everything easier to understand without having to dwell on the names of the fields themselves (mine have become very very very long). Finally, it would be even more beautiful if these descriptions written by us appeared later when we place the pointer on the relevant fields.
In the hope of having explained myself, I still post some screenshots of what I mean (LibreOffice Base).
Thanks in advance to all!

#### Is your feature request related to an issue? Please include the issue number.
I don't think so because I didn't find anything...

#### Does this feature exist in another product or project? Please provide a link.
Yes, it does: [LibreOffice Base](https://www.libreoffice.org/DISCOVER/BASE/).

#### Do you have a screenshot? Please add screenshots to help explain your idea.
![IMG(1)](https://user-images.githubusercontent.com/61233637/118797753-2f6d4a80-b89d-11eb-8b94-29bb1e9643d8.png)
![IMG(2)](https://user-images.githubusercontent.com/61233637/118797775-3300d180-b89d-11eb-91ec-a68f0220e8c0.png)

**Comment by chrisjlocke** [comment]
Good idea, and I think it's been suggested before.  The issue is how to implement this within SQLite.  You could add your own table to the database, but at some point the name might clash with something else.  Descriptions could be stored in the project file, but then you could attach another database, and it all goes to pot.

As a workaround, SQLite does preserve the comments put in the database schema, eg,
```
CREATE TABLE User
        -- A table comment
(
        uid INTEGER,    -- A field comment
        flags INTEGER   -- Another field comment
);
```

I believe DB4S preserves this too, so tries to keep these.

**Comment by PautassoEGiacomina** [comment]
> Good idea, and I think it's been suggested before. The issue is how to implement this within SQLite. You could add your own table to the database, but at some point the name might clash with something else. Descriptions could be stored in the project file, but then you could attach another database, and it all goes to pot.
> 
> As a workaround, SQLite does preserve the comments put in the database schema, eg,
> 
> ```
> CREATE TABLE User
>         -- A table comment
> (
>         uid INTEGER,    -- A field comment
>         flags INTEGER   -- Another field comment
> );
> ```
> 
> I believe DB4S preserves this too, so tries to keep these.

In fact I had read about the possibility offered by SQLite to add personal comments / notes to the code by inserting them after "--". However, trying to create a table on DB Browser for SQLite and adding comments to the related fields with this method, not only do I not see them when I subsequently hover over the mouse, but I no longer see them even when I later reopen the table to modify it :(

**Comment by chrisjlocke** [comment]
Yes, after playing around with it, DB4S doesn't like it, and does rebuild the schema, so they could be lost.  Understandable, I guess.  
I seem to remember someone suggesting a 'schema' table where you could manually add records for fields, but again, this isn't tied into DB4S, so you'd have to print it out to view it, etc - not overly ideal.

**Comment by PautassoEGiacomina** [comment]
> Yes, after playing around with it, DB4S doesn't like it, and does rebuild the schema, so they could be lost. Understandable, I guess.
> I seem to remember someone suggesting a 'schema' table where you could manually add records for fields, but again, this isn't tied into DB4S, so you'd have to print it out to view it, etc - not overly ideal.

I understand, thanks for the support anyway!! :)
